### PR TITLE
replace qf landing page image for gr15

### PR DESCRIPTION
**Pull request by schultztimothy**
PGF request to update image

Screen Shot 2022-08-19 at 4 44 49 PM

### CODE REVIEW

**Review by gdixon — APPROVED** [review]

File: app/grants/templates/grants/landing/landing_qf_active.html [hunk 1]
```diff
@@ -22,7 +22,7 @@ <h4 class="text-wrap gc-font-base font-weight-normal mb-md-5 mb-4 pt-md-4 pt-1 t
       {% if show_main_round_banner %}
         <img src="{% static 'v2/images/grants/gr14-banner.png' %}" class="d-block w-100"/>
       {% else %}
-        <img src="{% static 'v2/images/grants/rocket-right.svg' %}" height="241" width="311" />
+        <img src="{% static 'v2/images/grants/garden_qf_landing.png' %}" height="auto" width="100%" />
```
Comment (giliomeejg) [inline app/grants/templates/grants/landing/landing_qf_active.html:25]: garden_qf_landing.png is currently a 5.5Mb image.

Might be good to try and get this down to < 250Kb

Comment (giliomeejg) [reply app/grants/templates/grants/landing/landing_qf_active.html:25]: Found a tool that seems to preserve the transparency:  https://resizing.app/features/resize-png/

Below is one I've resized to width=600px:

garden_qf_landing

**Comment by lthrift** [comment]
i don't think it needs to block the deploy, but wondering if anyone from MMM has seen this? The image looks really small compared to the fonts next to it. Not what I'm used to seeing in this image space.

**Comment by giliomeejg** [comment]
@lthrift - I'll pull a local copy and review.

**Pull request merged**